hibernate-core/src/main/java/org/hibernate/cfg/AvailableSettings.java

```diff
@@ -2556,6 +2556,23 @@ public interface AvailableSettings {
 	@Incubating
 	String PREFERRED_INSTANT_JDBC_TYPE = "hibernate.type.preferred_instant_jdbc_type";
 
+	/**
+	 * Specifies the preferred JDBC type for storing basic array and collection values. When no
+	 * type is explicitly specified, a sensible
+	 * {@link org.hibernate.dialect.Dialect#getPreferredSqlTypeCodeForArray()
+	 * dialect-specific default type code} is used.
+	 *
+	 * Can be overridden locally using {@link org.hibernate.annotations.JdbcType},
+	 * {@link org.hibernate.annotations.JdbcTypeCode} and friends
+	 *
+	 * Can also specify the name of the {@link org.hibernate.type.SqlTypes} constant field.  E.g.
```

Comment:
Just a general thought as we now duplicate this exact sentence multiple times.  We should move this to somewhere central and refer to that from these field Javadocs.  

Not here, to be clear, as I was serious when I said I want to stop this new round of config explosion.  We worked hard in 6 dev to clean tons of settings and we REALLY need to be judicious here

Reply:
Going to be removed anyway